packages/mongodb-memory-server-core/src/util/getos/index.ts

```diff
@@ -125,9 +126,13 @@ async function getLinuxInfomation(): Promise<LinuxOS> {
  */
 async function tryLSBRelease(): Promise<LinuxOS | undefined> {
   try {
-    const lsb = await promisify(exec)('lsb_release -a'); // exec this for safety, because "/etc/lsb-release" could be changed to another file
+    // use upstream lsb file if it exists (like in linux mint)
+    if (fs.existsSync('/etc/upstream-release/lsb-release')) {
```

Comment:
if already testing for this file (that i have never seen before), maybe add an check for `/etc/lsb-release` before the command

Reply:
according to this [comment](https://github.com/nodkz/mongodb-memory-server/pull/424/files#diff-730c93549b867e9788e59c98aa3caf807c4b4d07012c9d2e2dc04b05ee32bf1eR134), that you already worked around by using lsb_release instead of the file itself...

Reply:
sorry, what comment? for me this link goes to somewhere random in the changes in the pr (without an line marking or any comment near other than codecov)

Reply:
hm. in the next line simply:
```
    // exec this for safety, because "/etc/lsb-release" could be changed to another file
```